dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/ProjectServiceImpl.java

```diff
@@ -855,4 +862,11 @@ public List<Long> getAuthorizedProjectCodes(User loginUser) {
                 .map(Project::getCode)
                 .collect(Collectors.toList());
     }
+
+    private void setRunningProcessInstanceNumber(List<Project> projectList) {
+        List<Long> projectsCodes = projectList.stream().map(Project::getCode).collect(Collectors.toList());
+        List<ProjectInstanceRunningDTO> projectInstanceRunningDTOList = projectMapper.queryRunningProcessInstanceNumber(projectsCodes);
```

Comment:
hi @chen877511127  i suggest that call method of queryRunningProcessInstanceNumber  when ```projectsCodes.size()>0```,
Otherwise, the SQL statement of the queryRunningProcessInstanceNumber method will not trigger the ```<if test="projectsCodes != null and projectsCodes.size() > 0"> ```, and all project information will be queried.

Reply:
I don't know what else I need to do? When can my submission be merged?